### PR TITLE
Replaced description by data format of various parser plugins #3087

### PR DESCRIPTION
**Pull request by joachimmetz**
Replaced description by data format of various parser plugins #3087

### CODE REVIEW

File: utils/export_supported_formats.py [hunk 1]
```diff
@@ -294,14 +275,29 @@ def _GetDataFormatInformationFromPackage(self, package):
...
+
+            dtfabric_file = os.path.join(package_path, ''.join([name, '.yaml']))
+            if os.path.exists(dtfabric_file):
+              definitions_registry = (
+                  dtfabric_registry.DataTypeDefinitionsRegistry())
+              definitions_reader = (
+                  dtfabric_reader.YAMLDataTypeDefinitionsFileReader())
+
+              try:
+                definitions_reader.ReadFile(definitions_registry, dtfabric_file)
```
Comment (rgayon) [inline utils/export_supported_formats.py:294]: I don't see any code using definitions_registry. Is this normal?

If another PR is coming, can you add an explicit TODO?

Comment (joachimmetz) [reply utils/export_supported_formats.py:294]: Ack, this is what I meant with `# TODO: determine url.` but agree that this is not very explicit ;) I'll update the TODO

Comment (joachimmetz) [reply utils/export_supported_formats.py:294]: updated the TODO

**Comment by joachimmetz** [comment]
@rgayon PTAL

**Review by rgayon — APPROVED** [review]
I see codefactor is not happy with the try/Exception. Maybe you want to ignore

**Comment by joachimmetz** [comment]
> I see codefactor is not happy with the try/Exception. Maybe you want to ignore

thx I'll ignore, I don't find a lot what codefactor flags actionable and therefore that helpful. The broad except is there to have the script continue, since this is a utility script the impact is minimal.

**Pull request merged**